db/migrate/20210614164900_change_email_to_citext_for_users.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeEmailToCitextForUsers < ActiveRecord::Migration[6.1]
```

Comment:
@malparty Since the migration to create users is not yet merged on the heroku database, it can be better to change the type in origination migration. On your local system, we can rollback the migration and run again

Reply:
All right! I'm on it! (I'll include the rename of last_name first_name too then) :) 
Thanks!

Reply:
Done! ;-)